Edits to Alerts section (#63)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_advanced-features.adoc

// Module included in the following assemblies:
//
// <List assemblies here, each on a new line>

// This module can be included from assemblies using the following include statement:
// include::<path>/con_advanced-features.adoc[leveloffset=+1]

// The file name and the ID are based on the module title. For example:
// * file name: con_my-concept-module-a.adoc
// * ID: [id='con_my-concept-module-a_{context}']
// * Title: = My concept module A
//
// The ID is used as an anchor for linking to the module. Avoid changing
// it after the module has been published to ensure existing links are not
// broken.
//
// The `context` attribute enables module reuse. Every module's ID includes
// {context}, which ensures that the module has a unique ID even if it is
// reused multiple times in a guide.
//
// In the title, include nouns that are used in the body text. This helps
// readers and search engines find information quickly.
// Do not start the title with a verb. See also _Wording of headings_
// in _The IBM Style Guide_.
[id="advanced-features_{context}"]

> The following features are optional and can greatly increase the efficacy of Service Telemetry Framework:

Reviewer: I don't like the word "efficacy" here since it's so easy to confuse with efficiency and I'm not sure it's the correct usage anyways. Will provide a suggestion.

Author: Yes, this is poor excuse for an intro to the advanced features section but we need to include something. Would be delighted if you could come up with something that we can work with!

> The following features are optional and can greatly increase the efficacy of Service Telemetry Framework:

Reviewer: ```suggestion
The following features are optional but can provide additional functionality to the Service Telemetry Framework:
```
